swiper/app/controllers/listings_controller.rb

```diff
@@ -172,6 +172,22 @@ def reserve
 
     def complete
         @listing = Listing.find(params[:id])
+        @seller = User.find(@listing.user_id)
+
+        params[:rating] = params[:rating].to_i
+        if params[:rating] > 5 || params[:rating] < 0
+            redirect_to listing_path(@listing, :id => params[:id]), :flash => { :error => "You submitted an invalid rating value!" } and return
+        else
+            puts @seller.rating
+            if @seller.rating == nil
+                new_rating = params[:rating]
+            else
+                new_rating = @seller.rating + params[:rating]
```

Comment:
shouldn't this be + params[:rating] / rating_count?

Reply:
Oh, are we keeping a running average? That makes sense why its a float

Reply:
or more accurately new rating = (@seller.rating + params[:rating]) / (rating_count + 1)

Reply:
No, that value would quickly converge to zero